copernicus_services/hrsi-fsc-ndsi/README.md

```diff
@@ -0,0 +1,24 @@
+# CLMS HR-S&I: Fractional Snow Cover - Normalised difference snow index (%) visualisation script
+
+<a href="#" id='togglescript'>Show</a> script or [download](script.js){:target="_blank"} it.
+<div id='script_view' style="display:none">
+{% highlight javascript %}
+{% include_relative script.js %}
+{% endhighlight %}
+</div>
+
+## General description of the script  
+This script visualises Fractional Snow Cover NDSI. The product provides the NDSI values in the detected snow area, i.e. area where
+FSCTOC > 0%.
+
+  
+## Description of representative images
+*23rd April 2020, Innsbruck, AT* 
+
+![NDSI](fig/figure.png)  
+
+## Resources
+
+- [Data source](https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-snow-and-ice-monitoring/snow-products)
+
+- Entry in public collections
```

Comment:
I would expect a link here, is that going to be added at a later time? If so I think it is better to add it when the public collections entry is available and not have some text pointing to nothing.